### ISSUE TITLE
header.html:157: Sort top menu subcategories by name #}

### ISSUE THREAD

**Issue by 0pdd** [fidals/shopelectro#974]
The puzzle `971-f15c5594` from #971 has to be resolved: 

https://github.com/fidals/shopelectro/blob/77991fd4612ae6a434e23aef3fd4d0ed83507740/templates/layout/header.html#L157-L157

The puzzle was created by duker33 on 28-Sep-19. 

Estimate: 60 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0pdd** [comment]
@artemiyds2 @duker33 please pay attention to this new issue.

**Comment by duker33** [comment]
at the prod catalog custom page is the parent for root category pages:
```
>>> [c.page.parent for c in  Category.objects.filter(parent=None)]
[<Page: catalog>, <Page: catalog>, <Page: catalog>, <Page: catalog>, <Page: catalog>, <Page: catalog>, <Page: catalog>,<Page: catalog>, <Page: catalog>, <Page: catalog>, <Page: catalog>, <Page: catalog>, <Page: catalog>]
```

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#980](https://github.com/fidals/shopelectro/issues/980), [#981](https://github.com/fidals/shopelectro/issues/981) are still not solved.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#981](https://github.com/fidals/shopelectro/issues/981) is still not solved; solved: [#980](https://github.com/fidals/shopelectro/issues/980).
